contracts/interfaces/yearn/IYearnVault.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
```

Comment:
Why not use `^0.8.10` and make it easier to upgrade to 0.8.14? :)

or if you prefer strict versions, why not upgrade to 0.8.14 anyway?

Reply:
don't really have an answer for this, the same statement is in all contract files in the project